sdk/lib/rtl/actctx.c

```diff
@@ -23,6 +23,21 @@
 
 #include <wine/unicode.h>
 
+#define GetProcessHeap() RtlGetProcessHeap()
+#define GetCurrentProcess() NtCurrentProcess()
+#define FIXME DPRINT1
+#define WARN DPRINT1
+#define TRACE DPRINT
+#define __TRY _SEH2_TRY
+#define __EXCEPT_PAGE_FAULT _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
+#define __ENDTRY _SEH2_END
+#define FILE_END_OF_FILE_INFORMATION FILE_STANDARD_INFORMATION
+#define FileEndOfFileInformation FileStandardInformation
+#define RELATIVE_PATH RtlPathTypeRelative
```

Comment:
Are these definitions added because Wine's user mode headers are not available here? Is this part of RTL actually compiling for kernel mode only? If not, then maybe we can include an appropriate the corresponding Wine header instead? E. g., wine/debug.h, wine/exception.h and so on?

Reply:
This is because Wine doesn't understand that RTL code (in ntdll and ntoskrnl) must not use higher-level functions from Win32.

Reply:
Ah, OK then. Thanks.

Reply:
This is user mode only. I'll have a look at whether we can include those headers.

Reply:
"wine/exception.h" works, "wine/debug.h" isn't that simple, as it needs quite a few additional changes and linking to libwine. This is something I would prefer to do in a different PR, because it has functional implications.

Reply:
> This is user mode only. I'll have a look at whether we can include those headers.

Well this is compiled _**in**_ ntdll, which would be better to be independent on inclusions of win32-only headers.
(I would say, an exception could be if these are hidden behind wine-specific headers like those you mentioned.)